po/es.po

```diff
@@ -88,7 +88,7 @@ msgstr "Firmware"
 
 #: lib/logitech_receiver/i18n.py:48
 msgid "Hardware"
-msgstr "Hardware"
+msgstr "Material"
```

Comment:
As a native Spanish speaker, I'd like to point out that the word "hardware" is pretty much used every day, and this is the first time I've seen "Material" as a translation of hardware. I'm pretty sure this'd be very confusing for users.